03-task-manager/starter/quizeAnswers.txt

```diff
@@ -0,0 +1,8 @@
+1 middleware functions have access to the objects req and res and also the next function. 
+we can modify req or res using middleware, that makes it a powerfull tool. 
+next function calling for the next function - always must end our middleware with next()
```

Comment:
Both true statement and most important ones. Good work. Middlewares are very important in node application and there are many other properties that you will learn las you learn from the documents and come across ideas when you work on project and other parts of the courses such authentication and authorization, testing etc. Keep up the good work. Good work ALena!